fero/analysis.py

```diff
@@ -242,6 +268,48 @@ def _make_col_name(col_name: str, cols: List[str]) -> str:
 
         return col_name
 
+    @staticmethod
+    def _s3_upload(inbox_response, file_name, fp) -> None:
+
+        files = {
+            "file": (
+                file_name,
+                fp,
+            )
+        }
+
+        res = requests.post(
+            inbox_response["url"],
+            data=inbox_response["fields"],
+            files=files,
+        )
+
+        if res.status_code != 204:
+            raise FeroError("Error Uploading File")
+
+    @staticmethod
+    def _azure_upload(inbox_response: dict, fp) -> None:
+        blob_client = BlobClient.from_blob_url(
+            f"https://{inbox_response['storage_name']}.blob.core.windows.net/{inbox_response['container']}/{inbox_response['blob']}?{inbox_response['sas_token']}"
+        )
+        blob_client.upload_blob(io.BytesIO(fp.read().encode()))
+
+    def _upload_file(self, fp: IO, file_tag: str, prediction_type: str) -> str:
```

Comment:
I would complain about the lack of docstrings, but....this is just cutting and pasting my code so 😉

Reply:
Lol - It is a `_` function

Reply:
I'll let Todd weigh -- happy to fill one out